### PR TITLE
navbar color fixed

### PR DESCRIPTION
**Pull request by tanishq-arya**
## What is the change?
1. Removed classes text-info and btn-info
2. Added class btnFrame to nav buttons
3. Commented excess css for nav-link

## Related issue?
close: #638 

## How was it tested?
Remove this line and tell us how the code was tested.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Did you lint your code before making the Pull Request?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Screenshot (1281)
Screenshot (1280)

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -114,9 +114,14 @@ body {
...
+.nav-link:hover {
+    color: var(--color-secondary) !important;
+    background-color: var(--color-text);
 }
 
-.navbar-right li a {
+/* .navbar-right li a {
     color: var(--color-primary) !important;
     font-weight: bold;
     margin: 0.4rem 0.6rem 0.3rem 0.6rem;
```
Comment (Pranav016) [inline public/css/styles.css:127]: You can remove all the commented code..

File: views/partials/header.ejs [hunk 2]
```diff
@@ -29,29 +29,29 @@
           <div id="navbar" class="collapse navbar-collapse">
             <ul class="navbar-nav ml-md-auto align-items-center">
               <li id="home" class="nav-item">
-                <a class="nav-link text-info" href="/">HOME</a>
```
Comment (Pranav016) [inline views/partials/header.ejs:32]: was this `text-info` class removed from `styles.css` file as well if it is not being used anywhere else ?

**Review by Pranav016 — CHANGES_REQUESTED** [review]
PR looks good, a couple changes

File: public/css/styles.css (hunk 1 above)
Comment (tanishq-arya) [reply public/css/styles.css:127]: ok

File: views/partials/header.ejs (hunk 2 above)
Comment (tanishq-arya) [reply views/partials/header.ejs:32]: This is a bootstrap class so it is not in styles.css

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level2/

**Pull request merged**